conf.py

```diff
@@ -303,7 +303,7 @@ def xapian_code_example_filename(ex):
     return "code/%s/%s%s" % (highlight_language, ex, ext)
 
 # Return the command to show in the generated docs.
-def xapian_code_example_command(ex):
+def xapian_code_example_command(ex, option):
```

Comment:
`option` with values `0` or `1` is rather cryptic.  Why not call the parameter `letor`, and make the values `False` and `True`?  Or pass in the `options` dict.